pom.xml

```diff
@@ -338,7 +342,6 @@
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>3.3.2</version>
           <configuration>
-            <source>8</source>
```

Comment:
The [default value](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#source) is `${maven.compiler.source}`, which we are setting above.